src/SFML/Window/WindowImpl.cpp

```diff
@@ -175,35 +176,10 @@ void WindowImpl::setMaximumSize(const std::optional<Vector2u>& maximumSize)
 
 
 ////////////////////////////////////////////////////////////
-Event WindowImpl::popEvent(bool block)
+Event WindowImpl::popEventImpl()
```

Comment:
Why popEvent**Impl**?

If you think it's too confusing with the other names, I'd recommend taking something longer than just appending `Impl`.

I also recommend moving the function below the other two functions.

Reply:
I'll move the function and see if I can come up with a better name. Suggestions?